src/main/java/Cleese/Storage.java

```diff
@@ -0,0 +1,50 @@
+package Cleese;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+public class Storage {
+    private static String filePath;
+    private static File file;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+        this.file = new File(filePath);
+    }
```

Comment:
The usage of `this.` keyword. It is recommended to use the `this` keyword only when a field is shadowed by a method or constructor parameter.

![image](https://user-images.githubusercontent.com/36361212/152059587-025c57ef-25c7-4b34-8f0e-4e1c87d32fb0.png)